spec/models/imported_clients_csv_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
```

Comment:
Can we add ~5 rows to the sample files so we don't have an N of 1 (those often have their own sets of issues.) 

Also, let's add a test where it matches an existing ImportedClient so we can make sure that's working. 

Finally, a test that verifies that the people we expect are the people we actually get (compare emails maybe?), especially when one already exists.